hadoop-hdfs-project/hadoop-hdfs-httpfs/src/main/java/org/apache/hadoop/fs/http/client/HttpFSFileSystem.java

```diff
@@ -1773,6 +1775,17 @@ public FsStatus getStatus(final Path path) throws IOException {
     return JsonUtilClient.toFsStatus(json);
   }
 
+  public Collection<ErasureCodingPolicyInfo> getAllErasureCodingPolicies() throws IOException {
+    Map<String, String> params = new HashMap<>();
+    params.put(OP_PARAM, Operation.GETECPOLICIES.toString());
+    Path path = new Path(getUri().toString(), "/");
+    HttpURLConnection conn =
+        getConnection(Operation.GETECPOLICIES.getMethod(), params, path, true);
```

Comment:
do we need this makeQualified to be ``true`` here? we hardcoded / itself here, getAllStoragePolicies seems to pass ``false`` as well

Reply:
Thank you for your valuable suggestion. I sincerely appreciate it and will promptly implement the required adjustments to the code!